### PR TITLE
Address comments from previous pull request

### PR DESCRIPTION
**Pull request by sangttruong**

### CODE REVIEW

File: R/package.R [hunk 1]
```diff
@@ -10,5 +10,7 @@
 # The following block is used by usethis to automatically manage
 # roxygen namespace tags. Modify with care!
 ## usethis namespace: start
+#' @importFrom hyperSpec wl_convert_units
```
Comment (GegznaV) [inline R/package.R:13]: This is OK as you did. I just wanted to let you know that the following syntax is also valid (package name and the names all imported functions; may also span several lines):

```
#' @importFrom hyperSpec wl_convert_units .spc_io_postprocess_optional
```

File: DESCRIPTION [hunk 2]
```diff
@@ -1,11 +1,12 @@
 Package: hySpc.read.spe
 Type: Package
 Version: 0.0.0.9000
-Title: File Import Functions for Spectra produced by Princeton Instruments spectrometers
+Title: File Import Functions for Spectra Produced by Princeton Instruments Spectrometers
```
Comment (GegznaV) [inline DESCRIPTION:4]: OK

File: DESCRIPTION [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
 Authors@R: c(
     person("Claudia",   "Beleites",  role = c("aut"), email = "Claudia.Beleites@chemometrix.gmbh"),
     person("Sang",     "Truong", role = c("aut","cre"), email = "sttruong@stanford.edu"),
```
Comment (GegznaV) [inline DESCRIPTION:7]: 1. If Sang will be the maintainer of this package (`cre`), then I'm not sure if I have to be included. Yet, as I review the code, I should be included as a contributor (`ctb`) and not as an author (`aut`). I did not create any SPE-related functions.

2. The order of authors (in my opinion) should go in this order (by the amount of contribution) Claudia and Roman, then Sang, then others.

**Review by GegznaV — CHANGES_REQUESTED** [review]

File: DESCRIPTION (hunk 3 above)
Comment (GegznaV) [reply DESCRIPTION:7]: @sangttruong, when you update this file, I think, this portion of changes could be merged.

Comment (sangttruong) [reply DESCRIPTION:7]: It's good to go now @GegznaV

**Pull request merged**